pages/account/statistics.js

```diff
@@ -1,5 +1,6 @@
 import { authOptions } from "../api/auth/[...nextauth]";
 import { unstable_getServerSession } from "next-auth/next";
+import ProfileProgress from "../../components/statistics/ProfileProgress"
```

Comment:
a note for next time, we use `@components/` for imports now

Reply:
Got it!